pennylane/fermi/conversion.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Utility functions to convert between ``~.FermiWord`` and other PennyLane formats."""
+
+from functools import singledispatch
+from typing import Union
+
+from pennylane.pauli import PauliWord, PauliSentence
+from .fermionic import FermiWord, FermiSentence
+
+
+@singledispatch
+def jordan_wigner(fermi_operator: (Union[FermiWord, FermiSentence]), ps=False) -> PauliSentence:
```

Comment:
We should support custom wire maps. If this PR is already large, we can have a separate PR for it.

Reply:
I think its fine to add here, shouldn't add much

Reply:
Added, but see comment below for question about implementation